src/components/AffiliationsManager/AffiliationsManager.test.js

```diff
@@ -86,27 +86,21 @@ describe('AffiliationsManager', () => {
         await userEvent.click(assignmentCheckboxes[2]);
         await userEvent.click(await screen.findByText('ui-users.affiliations.manager.filter.assignment.assigned'));
 
-        expect(await screen.findAllByRole('row')).toHaveLength((affiliations.length - 3) + 1);
+        expect(await screen.findAllByRole('row')).toHaveLength((affiliations.length - 3));
 
         await userEvent.click(await screen.findByLabelText(/Clear selected filters for/));
 
-        expect(await screen.findAllByRole('row')).toHaveLength(affiliations.length + 1);
+        expect(await screen.findAllByRole('row')).toHaveLength(affiliations.length);
       });
 
       it('should reset search and filters when \'Reset all\' button was clicked', async () => {
```

Comment:
This test is deprived of semantic checkpoints after the expected results are thrown out.